lib/creek.rb

```diff
@@ -3,12 +3,14 @@
 require 'creek/cell_value_extractor'
 require 'creek/styles/constants'
 require 'creek/styles/style_types'
-require 'creek/styles/converter'
 require 'creek/styles'
 require 'creek/cell_styles'
 require 'creek/sheet'
 require 'creek/shared_strings'
 
+Dir[File.dirname(__FILE__) + '/creek/styles/value_convert/*.rb'].each {|file| require file }
```

Comment:
Use `File.join` instead of `+`. https://ruby-doc.org/core-3.0.2/File.html#method-c-join